doc/classes/EditorImportPlugin.xml

```diff
@@ -124,17 +124,18 @@
 			<return type="bool" />
 			<description>
 				Tells whether this importer can be run in parallel on threads, or, on the contrary, it's only safe for the editor to call it from the main thread, for one file at a time.
-				If this method is not overridden, it will return [code]false[/code] by default.
 				If this importer's implementation is thread-safe and can be run in parallel, override this with [code]true[/code] to optimize for concurrency.
+				If not overridden, it will return [code]false[/code] by default.
```

Comment:
This is more so for consistency.
```suggestion
				If not overridden, returns [code]false[/code].
```
Alternatively I don't see anything wrong with the prior sentence.
```suggestion
				If this method is not overridden, it will return [code]false[/code].
```